deployment/bicep/gateway/appgw.bicep

```diff
@@ -25,9 +25,8 @@ param appGatewayCertificateData     string
 param keyVaultName                  string
 param keyVaultResourceGroupName     string
 
-var namingStandard          = '${appGatewayName}-prod-${location}-001'
```

Comment:
Should `prod` be abstracted as an `environment` string variable? Looks like it could be passed from `main.bicep`.